content/plugins/_index.en.md

```diff
@@ -3,10 +3,79 @@ title: Plugins
+## How to install a plugin
+
+### Install using Plugins Admin
+The Plugins Admin allows you to easily install plugins that are in the Plugins List.  To do so, place a check mark next to the Plugin(s) you wish to install, then select Install.
+
+### Install plugin manually
+If the plugin you want to install is not listed in the Plugins Admin, you may still install it manually.  The plugin (in the DLL form) should be placed in the \plugins subfolder of the Notepad++ Install Folder, under the subfolder with the same name of plugin binary name without file extension.
+For example, if the plugin you want to install named `myAwesomePlugin.dll`, you should install it with the following path:
+`%PROGRAMFILES(x86)%\Notepad++\plugins\myAwesomePlugin\myAwesomePlugin.dll`
+
+Once you installed the plugin, you can use (and you may configure) it via the menu "Plugins".
+
+## How to develop a plugin
+
+### Getting started - Make your plugin in less 10 minutes (C++)
+
+Here are the instructions to make your first Notepad++ plugin in less 10 minutes, by following 6 steps:
+
+1. Download and unzip [Notepad++ Plugin Template](https://github.com/npp-plugins/plugintemplate/archive/v3.zip).
+2. Open NppPluginTemplate.vcproj in your Visual Studio.
+3. Define your plugin name in PluginDefinition.h
+4. Define your plugin commands number in PluginDefinition.h
+5. Customize plugin commands names and associated function name (and the other stuff, optional) in PluginDefinition.cpp.
+6. Define the associated functions.
+You are guided by the following comments in both [PluginDefinition.h](https://github.com/npp-plugins/plugintemplate/blob/master/src/PluginDefinition.h) and [PluginDefinition.cpp](https://github.com/npp-plugins/plugintemplate/blob/master/src/PluginDefinition.cpp) files:
+
+```
+//-- STEP 1. DEFINE YOUR PLUGIN NAME --//
+//-- STEP 2. DEFINE YOUR PLUGIN COMMAND NUMBER --//
+//-- STEP 3. CUSTOMIZE YOUR PLUGIN COMMANDS --//
+//-- STEP 4. DEFINE YOUR ASSOCIATED FUNCTIONS --//
+```
+A good sample illustrates better the whole picture than a detailed documentation. You can check [Notepad++ Plugin Demo](https://github.com/npp-plugins/plugindemo/releases) to learn how to make some commands more complex. 
+
+However, the knowledge of Notepad++ plugin system is required, if you want to accomplish some sophisticated plugin commands.
+
+You can use [Plugin development forum](https://notepad-plus-plus.org/community/category/5/plugin-development) for any technical questions/answers and the announcement your new plugin.
+
+
+### In other languages
+
+* [Delphi](https://sourceforge.net/projects/npp-plugins/files/DelphiPluginTemplate/DelphiPluginTemplate%202.0%20UNICODE/DelphiPluginTemplate2.zip/download)
+* [C#](http://sourceforge.net/projects/sourcecookifier/files/other%20plugins/NppPlugin.NET.v0.5.zip/download)
```

Comment:
Current update therefore is available at https://github.com/kbilsted/NotepadPlusPlusPluginPack.Net